src/components/Accordion/Accordion.spec.tsx

```diff
@@ -0,0 +1,74 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import * as React from 'react';
+import { Accordion } from './Accordion';
+
+describe('Accordion', () => {
+    it('allows rendering react nodes as children', () => {
+        render(
+            <Accordion expanded>
+                <button>button</button>
+                <p>paragraph</p>
+            </Accordion>
+        );
+        expect(screen.getByText('button')).toBeInTheDocument();
+        expect(screen.getByText('paragraph')).toBeInTheDocument();
+    });
+
+    it('render default variant closed', () => {
+        render(
+            <Accordion heading="Some heading" info="some info" description="some description" buttonLabel="button">
+                <p>Place holder</p>
+            </Accordion>
+        );
+        expect(screen.getByText('Some heading')).toBeInTheDocument();
+        expect(screen.getByText('some info')).toBeInTheDocument();
+        expect(screen.getByText('some description')).toBeInTheDocument();
+        expect(screen.getByText('button')).toBeInTheDocument();
```

Comment:
Suggestion: We could add one more assertion that ensures that the `Place holder` is also NOT in the document.

Reply:
Done